client/version.go

```diff
@@ -7,15 +7,40 @@ import (
 	"github.com/moby/moby/api/types"
 )
 
-// ServerVersion returns information of the docker client and server host.
-func (cli *Client) ServerVersion(ctx context.Context) (types.Version, error) {
+// ServerVersionOptions specifies options for the server version request.
+type ServerVersionOptions struct {
+	// Currently no options are supported.
+}
+
+// ServerVersionResult contains information about the Docker server host.
+type ServerVersionResult struct {
+	// Platform is the platform the server is running on.
+	Platform string
```

Comment:
We should keep platform as a struct; it was designed to have a single field (for now), but to allow to be extended.